apps/src/aichat/redux/aichatRedux.ts

```diff
@@ -125,12 +128,13 @@ export const publishModel = createAsyncThunk(
     dispatch(setModelCardProperty({property: 'isPublished', value: true}));
 
     const rootState = thunkAPI.getState() as RootState;
-    const {currentAiCustomizations, savedAiCustomizations, chatMessages} =
-      rootState.aichat;
+    const {currentAiCustomizations, savedAiCustomizations} = rootState.aichat;
+    const levelId = rootState.progress.currentLevelId;
     await saveAiCustomization(
       currentAiCustomizations,
       savedAiCustomizations,
-      chatMessages,
+      'PUBLISH_MODEL_CARD_INFO',
```

Comment:
I was wondering if we could do the thing where we pass in the event via `EVENTS[PUBLISH_MODEL_CARD_INFO]` rather than having these strings floating around? But I think you'd probably then just have a generic `string` type for your `eventKey` param (or three really long strings), which is maybe not ideal. 🤷

Reply:
Yeah - it looks like I'll need to type as `string` which I agree isn't ideal. But I'll go ahead and use the value (event description) instead of the string literals. Thanks!